src/actions.js

```diff
@@ -231,18 +231,18 @@ export function newMedicalItem() {
 
 export function fetchMedicalServiceMutation(mm, clientMutationId) {
   const payload = formatPageQuery(
-    "mutationLogs",
+    "medicalServices",
     [`clientMutationId:"${clientMutationId}"`],
-    ["id", "medicalServices{id}"],
+    ["uuid"],
```

Comment:
Are you sure this is not used by other modules like claim etc? It seems to me that something could brake in other modules.

Reply:
This is used only here. Moreover, this was never working properly. There is no argument like _medicalServices_ on _mutationLogs_ query.